eagle-core/eagle-alert-parent/eagle-alert-app/src/main/resources/META-INF/providers/org.apache.eagle.alert.app.AlertUnitTopologyAppProvider.xml

```diff
@@ -25,6 +25,12 @@
     <configuration>
        <!-- alert topology sizing parameters -->
         <property>
+            <name>topology.name</name>
+            <displayName>topology.name</displayName>
+            <value>alertUnitTopology_1</value>
```

Comment:
In https://github.com/apache/incubator-eagle/commit/fc2407cd53438a31b4c1072ce7d5b08578b8e43d, it will install topology metadata automatically including topology name.